mantle/kola/tests/rhcos/luks.go

```diff
@@ -68,7 +50,6 @@ func init() {
 		Name:                 `rhcos.luks.tang`,
 		Flags:                []register.Flag{},
 		Distros:              []string{"rhcos"},
-		Platforms:            []string{"qemu-unpriv"},
```

Comment:
These `ClusterSize: 0` tests are really "coreos-assmber local tests" - we still require a qemu image and that should be reflected somewhere.

Why did you remove this?  Was it breaking something?

Reply:
Aside: as of recently anything using TPM2 can also run in GCP: https://github.com/openshift/installer/pull/2921